co2eq/energy/energyfootprints.yml

```diff
@@ -0,0 +1,191 @@
+heating:
+  coal boiler:
```

Comment:
all those keys need to correspond to the value of HEATING_SOURCE_.... in definitions.js.
I therefore suggest renaming all values in definitions.js such that the value is equal to the name:
- instead of `export const HEATING_SOURCE_COAL_BOILER = 'coalBoiler';`, write `export const HEATING_SOURCE_COAL_BOILER = 'HEATING_SOURCE_COAL_BOILER';` in definitions.js 
- in this file, instead of having `coal boiler:`, use `HEATING_SOURCE_COAL_BOILER` so they can be matched together.